pkgs/by-name/ta/tabby-terminal/package.nix

```diff
@@ -0,0 +1,215 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  git,
+  patch-package,
+  electron_29,
+  prefetch-yarn-deps,
+  fixup-yarn-lock,
+  python3,
+  makeWrapper,
+  http-server,
+  fontconfig,
+  pkg-config,
+  libsecret,
+  desktop-file-utils,
+}:
+
+let
+  tabby = rec {
+    pname = "tabby-terminal";
+    version = "1.0.215";
+
+    src = fetchFromGitHub {
+      owner = "Eugeny";
+      repo = "tabby";
+      rev = "v${version}";
+      hash = "sha256-l3Hyt3e1edJjb7vL66JDllkpHwTU5efsNbgJ5Pcvn60=";
+      leaveDotGit = true;
+      deepClone = true;
+    };
+
+    meta = with lib; {
+      description = "A terminal for a more modern age";
+      homepage = "https://tabby.sh";
+      license = licenses.mit;
+      mainProgram = "tabby";
+      maintainers = with maintainers; [ geodic ];
+      platforms = [ "x86_64-linux" ];
+    };
+
+    # We need to fix the argv logic as tabby only handles it for the node executable, but we are running it with electron
+    patches = [ ./fix-argv-prefix-splice.patch ];
+
+    electronVersion = "29.4.6";
+
+    electronHeaders = fetchurl {
+      url = "https://www.electronjs.org/headers/v${electronVersion}/node-v${electronVersion}-headers.tar.gz";
+      hash = "sha256-e0qiEGI/1rWK72oLXfKtbbc8KmOGMhMx4IbT6MGAjms=";
+    };
+
+    electronHeadersSHA = fetchurl {
+      url = "https://www.electronjs.org/headers/v${electronVersion}/SHASUMS256.txt";
+      hash = "sha256-0mCnKIR3GS1TL+g0h+y2cAlqbBogf88WxN8xJvmFl4g=";
+    };
+
+    buildInputs = [
+      nodejs
+      python3
+      fontconfig
+      electron_29
```

Comment:
Is there any way to make nix tooling ignore the fact that Tabby depends on an older version of electron?

Reply:
Find a way to only build the bundle then makeWrapper the bundle using one of the packaged electron. If it works it works.